filterexpr/filterexpr.go

```diff
@@ -0,0 +1,32 @@
+package filterexpr
```

Comment:
I don't understand how dep is going to find the generated file when you go to import this.
I think thats what one of these is for but im not sure if that will work 🤔
```
//go:generate make gen
```

Reply:
thats why im doing it in makefile
i could use go generate but not worth it for this case